amazon-s3/5.7/modules/ROOT/pages/amazon-s3-connector-studio.adoc

```diff
@@ -1,72 +1,102 @@
-. Configure the connector.
+To add and configure a connector in Studio:
 
-== Add the Connector to Your Project
+. <<create-mule-project,Create a Mule project>>.
+. <<add-connector-to-project,Add the connector to your Mule project>>.
+. <<configure-source,Configure a source for the connector's flow>>.
+. <<add-connector-operation,Add a connector operation to the flow>>.
+. <<configure-global-element,Configure a global element for the connector>>.
+. <<configure-other-fields,Configure the Other Connector Fields>>.
 
-Anypoint Studio provides two ways to add the connector to your Studio project: from the Exchange button in the Studio taskbar or from the Mule Palette view.
+When you run the connector, you can view the app log to check for problems in real time, as described in <<view-app-log,View the App Log>>.
 
-=== Add the Connector Using Exchange
+If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector]. If, after reading this topic, you need additional information about the connector fields, see the xref:amazon-s3-connector-reference.adoc[Amazon S3 Reference].
 
-. In Studio, create a Mule project.
-. Click the Exchange icon *(X)* in the upper-left of the Studio task bar.
-. In Exchange, click *Login* and supply your Anypoint Platform username and password.
-. In Exchange, search for "s3".
-. Select the connector and click *Add to project*.
-. Follow the prompts to install the connector.
+[[create-mule-project]]
+== Create a Mule Project
 
-=== Add the Connector in Studio
+In Studio, create a new Mule project in which to add and configure the connector:
 
-. In Studio, create a Mule project.
-. In the Mule Palette view, click *(X) Search in Exchange*.
-. In *Add Modules to Project*, type "s3" in the search field.
-. Click this connector's name in *Available modules*.
+. In Studio, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add Amazon S3 to your Mule project to automatically populate the XML code with the connector's namespace and schema location and to add the required dependencies to the project's `pom.xml` file:
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In the *Add Dependencies to Project* window, type `amazon s3` in the search field.
+. Click *Amazon S3* in *Available modules*.
 . Click *Add*.
 . Click *Finish*.
 
-== Configure an Input Source
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+[[configure-source]]
+== Configure a Source
+
+A source initiates a flow when a specified condition is met.
+You can configure one of these sources to use with Amazon S3:
 
-Configure an input source for Amazon S3 Connector, such as a connector operation, HTTP connector, or scheduler.
+* HTTP Listener
++
```

Comment:
Ideally, if there are connector-specific sources like this connector has, we document one of the sources step-by-step instead of the HTTP Listener source. This is a nice-to-have thing but not a high priority, so you can create a story for this in case someone has time in the future to do this.